src/editor/actions/actions.js

```diff
@@ -177,15 +131,12 @@ export function clearVariables() {
   };
 }
 
-export function changePageTitle(title, autosaveChanges = true) {
+export function updateTitle(title) {
```

Comment:
Removing the link to autosave from this function helps us avoid import cycles (since we don't need to link to autosave code inside whatever holds this function). It turns out that the number of cases where we actually want to save a title change are actually relatively rare, so it makes sense to handle that specially IMO.